api/proto/teleport/accesslist/v1/accesslist.proto

```diff
@@ -32,6 +32,9 @@ message AccessList {
 
   // spec is the specification for the access list.
   AccessListSpec spec = 2;
+
+  // access_list_status contains dynamically calculated fields.
+  AccessListStatus access_list_status = 3;
```

Comment:
Is there any reason why we aren't calling this `status` as specified by RFD 153?
```suggestion
  AccessListStatus status = 3;
```

Reply:
https://github.com/gravitational/teleport/pull/38650#discussion_r1503522744 This is why this was done (cc @tigrato)

Reply:
I meant renaming the proto message, not the proto field. Sorry if I created confusion. 

My goal was  AccessListStatus status = 3;

Reply:
Ahhh, no worries. I'll fix!